examples/analytics_specs/README.md

```diff
@@ -0,0 +1,46 @@
+# atlas-basic - cluster with analytics specs
+
+This example pretends to show how the developer can use the atlas-basic module to create a cluster defining its analytics_specs.
```

Comment:
```suggestion
This shows how the atlas-basic module can be used to create a cluster defining analytic nodes.
```
Would go for a more direct description, same phrasing can be applied for other examples if you see fit.